get_iplayer.cgi

```diff
@@ -4252,7 +4249,7 @@ sub insert_javascript {
 				tab.style.visibility = 'visible';
 				option.value = 'yes';
 				//button.innerHTML = '- ' + button.innerHTML.substring(2);
-				button.style.color = '#F54997';
+				//button.style.color = '#F54997';
```

Comment:
I was just matching the adjacent coding style. :)

Happy to delete all commented out code in that area, though.